frog/imports/ui/StudentView/index.js

```diff
@@ -1,28 +1,31 @@
 // @flow
 
 import React from 'react';
+import { TimeSync } from 'meteor/mizzao:timesync';
 import { Meteor } from 'meteor/meteor';
 import { createContainer } from 'meteor/react-meteor-data';
 
 import SessionBody from './SessionBody';
 import SessionList from './SessionList';
 import { Sessions } from '../../api/sessions';
 
-const StudentView = ({ user, sessions }) => {
+const StudentView = ({ user, sessions, currentTime }) => {
   const curSession = user.profile
     ? Sessions.findOne(user.profile.currentSession)
     : null;
   return (
     <div id="student" style={{ width: '100%', height: '100%' }}>
       {curSession
-        ? <SessionBody session={curSession} />
+        ? <SessionBody session={curSession} currentTime={currentTime} />
         : <SessionList sessions={sessions} />}
     </div>
   );
 };
 
 export default createContainer(() => {
+  if (TimeSync.serverOffset() > 500) TimeSync.resync();
```

Comment:
Why this? Did you see this in the documentation? Isn't serverOffset static and basically just means the difference between server and client? Nothing wrong with that in itself? As long as its stable...